helm/harbor/Chart.yaml

```
@@ -1,14 +1,14 @@
 apiVersion: v1
 name: harbor
-version: [[ .Version ]]
-appVersion: 1.10.7
+version: 1.4.0-dev
```

This needs to stay how it was previously for our CI.
```suggestion
version: [[ .Version ]]
```